Methods_Results_Summary.Rmd

```diff
@@ -46,47 +73,92 @@ tm_basemap("CartoDB.Positron") +
 
 ```
 
+:"Area of interest for this analysis in purple, consisting of a portion the
+western extent of Lake Superior, the Apostle Islands, and Chequamegon Bay."
+
 # Methods
 
 ## eePlumB
```

Comment:
It would be great to get some screenshots in here!

Reply:
totes!! I'm going to email about the time I have available for the next few weeks - we've got a deadline coming up that I need to focus on so deep work time is going to be limited on this project until after the 14th.